website/docs/reference/widgets/select.md

```diff
@@ -127,15 +127,44 @@ Style properties allow you to change the look and feel of the widget.
 | -------------------- | -------------------------------------------------------- |
 | **Label Text Color** | Allows you to set text color for the label.              |
 | **Label Text Size**  | Allows you to set the size of the label.                 |
-| **Label Font Style** | Allows you to choose a font style, i.e., bold or italic. |
+| **Label Font Style** | Allows you to choose a font style, such as bold or italic. |
 | **Border Radius**    | Allows you to define curved corners.                     |
 | **Box Shadow**       | Allows you to choose from the available shadow styles.   |
 
-### Server-side Filtering
+### Server-side filtering
 
-The Select widget has the option to configure server-side filtering, where search queries are sent to the backend, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value is not present in the default options.
+The Select widget has the option to configure server-side filtering, where search queries are sent to the backend, and responses are used to populate options on the Select widget. When enabling server-side filtering in the widget, please update the default value to contain both `label` and `value` in this format `{"label":<label>, "value": <value>}` if the default value isn't present in the default options.
 
 The video below explains how to configure this.
```

Comment:
nitpick: line 138 and line 142 seem to be repetitive. Let's just keep one of them.

Reply:
I can still see the same lines -
`The video below explains how to configure this.`
Video
`s shown in the video above, follow these steps to configure server side filtering:`

Reply:
Not sure why, on my end github shows the line `The video below explains how to configure this.` as removed and the review comment as "Outdated".

I need to make another couple of small changes, we'll see what it looks like after another push.